src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -127,5 +132,26 @@ protected void clearWireTransferValues(CreditMemoWireTransfer cmWireTransfer) {
     }
     
    // end KFSUPGRADE-779
-    
-  }
+
+    @SuppressWarnings("deprecation")
+    protected boolean validateCheckStubLength(VendorCreditMemoDocument document) {
+        boolean result = true;
+        CuCheckStubService cuCheckStubService = SpringContext.getBean(CuCheckStubService.class);

```

Comment:
similar thought here about service getters

Reply:
I'll adjust this to use an appropriate getter.